source/resource/config/hidden.php

```diff
@@ -1,9 +1,16 @@
 <?php
-// File for configuraions that shouldn't be shared with the user
-$hidden = array(
-  'fritzbox' => array( 'uri' => 'https://192.168.0.1:49443/', 'user' => 'CometVisuTestUser', 'pass' => 'cvtu4here' ),
-  //'fritzbox' => array( 'uri' => 'http://192.168.0.1:49000/', 'user' => 'CometVisuTestUser', 'pass' => 'cvtu4here' ),
-  //'fritzbox' => array( 'uri' => 'https://fritz.box:49443/', 'user' => 'CometVisuTestUser', 'pass' => 'cvtu4here' , 'selfsigned' => 'true'),
-  'influx' => array('uri' => 'https://172.17.0.1/proxy/ts/query', 'user' => 'docker', 'pass' => 'cwkvJPQeFlGlRKc402CA', 'selfsigned' => 'true')
-);
-?>
\ No newline at end of file
+// File for configurations that shouldn't be shared with the user
```

Comment:
Add marker to show `hidden.php` type version.

E.g.:
```
<?php
// hidden.php type version: 2
// File for configurations that shouldn't be shared with the user
...
```

Reply:
Both backends can read the old version, but save the new one. So any modified data will be automatically converted into the new structure on the first change over the REST API.
No other service should directly modify the internal data structure of the hidden.php, thats like writing to an private variable => nogo.
Write operations should always use the REST API. Is that possible with the Timberwolf?